lib/screens/v2/candidate_generator/bus_shelter.ex

```diff
@@ -45,11 +45,18 @@ defmodule Screens.V2.CandidateGenerator.BusShelter do
       header_instances(config, now, fetch_stop_name_fn),
       departures_instances_fn.(config),
       footer_instances(config),
-      placeholder_instances()
+      placeholder_instances(),
+      subway_status_instances(config)
     ]
     |> List.flatten()
   end
 
+  defp subway_status_instances(config) do
+    route_ids = ["Blue", "Orange", "Red", "Green-B", "Green-C", "Green-D", "Green-E"]
+    alerts = Screens.Alerts.Alert.fetch(%{route_ids: route_ids})
```

Comment:
FYI: `Alert.fetch` returns `[]` when the API query for alerts failed. I changed that in #891, and I think it would make sense to handle failures here since we might incorrectly report the status as "all clear" if alert data is unavailable for some reason.

Feel free to make a followup task for now, to create some sort of subway status no-data-state design.